src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -181,6 +183,48 @@ namespace Utils
             }
         }
 
+        /**
+         * @brief Auxiliary function to delete all key-value pairs from the database.
+         * @param columnHandle Column handle to delete from.
+         */
+        void deleteAllFromColumn(rocksdb::ColumnFamilyHandle* columnHandle)
```

Comment:
One last change, make it private. Given that it has a raw pointer as an argument it makes it susceptible to errors when calling it from the outsied

Reply:
Done!